src/data/markdown/docs/01 guides/02 Using k6/19 Javascript Compatibility Mode.md

```diff
@@ -4,8 +4,6 @@ excerpt: ''
 hideFromSidebar: true
 ---
 
-> ### 🎉 This feature is new as of version 0.26.0
```

Comment:
Do we want to delete these types of notes? I think they're useful to keep indefinitely as reference.

Reply:
Makes sense. However, I think it would make sense to incorporate them into the introduction in that case, as we'd otherwise get 🎉 's and alert boxes on every page given enough time 😅 .

Reply:
Incorporated it into the text instead.